apiary.apib

```diff
@@ -3047,6 +3050,33 @@ owner or have edit permissions.
 
 + Response 204
 
+## Add default subject link [/collections/{id}/links/default_subject]
+Links a default subject to a collection. This subject's first media
+location URL will be included in the serialized collection and used
+as the thumbnail. Update this attribute with `null` to use the first
```

Comment:
Or if you mean the route, it's handled here

https://github.com/zooniverse/Panoptes/pull/2296/files#diff-21497849d8f00507c9c8dcaf6288b136R124

and setting is allowed via this:
https://github.com/zooniverse/Panoptes/pull/2296/files#diff-e5f000780a616ca07cd53cdbbfdbfd59R14